lib/caffeine.rb

```diff
@@ -9,7 +9,7 @@
 require 'closure_tree'
 require 'draper'
 require 'refile/rails'
-require 'refile/image_processing'
+require 'refile/mini_magick'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.